CRM/Utils/Mail.php

```diff
@@ -183,14 +183,20 @@ public static function send(array &$params): bool {
       return FALSE;
     }
 
-    $textMessage = $params['text'] ?? NULL;
-    $htmlMessage = $params['html'] ?? NULL;
-    $attachments = $params['attachments'] ?? NULL;
-
-    // CRM-6224
-    if (trim(CRM_Utils_String::htmlToText($htmlMessage)) == '') {
+    $htmlMessage = $params['html'] ?? FALSE;
+    if (trim(CRM_Utils_String::htmlToText((string) $htmlMessage))) {
```

Comment:
I think this should be `== ''` like before, otherwise it always sets htmlMessage to false.

Reply:
opps - yes!